### PR TITLE
Feature/map

### PR DESCRIPTION
**Pull request by tobicorradi**
## Pull request type

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the new behavior?

- User can now see the map once the log in it's completed.
## Other information

- Stack: React, Typescript, React Router, React Hook Form, Axios, Yup, Tailwind css and Redux Toolkit

## Preview

Screen Shot 2022-05-20 at 13 08 57

Screen Shot 2022-05-20 at 13 11 56

### CODE REVIEW

File: src/components/common/Map/index.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  MapContainer, Marker, Popup, TileLayer,
+} from 'react-leaflet';
+import 'leaflet/dist/leaflet.css';
+
+function Map() {
+  const TOKEN = import.meta.env.VITE_MAP_ACCESS_TOKEN;
+  return (
+    <MapContainer center={[51.505, -0.09]} zoom={17} scrollWheelZoom>
+      <TileLayer
+        url={`https://api.mapbox.com/styles/v1/tobicorradi/cl3el2qne000o15pkcygzhmbq/tiles/{z}/{x}/{y}?access_token=${TOKEN}`}
```
Comment (juanfopaez) [inline src/components/common/Map/index.tsx]: Save this url into a const an reuse it here :) 

probably if you create a constants folder you can save it there

File: .env [hunk 2]
```diff
@@ -1 +1,2 @@
-VITE_API_URL=http://target-mvd-api.herokuapp.com/api/v1
\ No newline at end of file
+VITE_API_URL=http://target-mvd-api.herokuapp.com/api/v1
+VITE_MAP_ACCESS_TOKEN=pk.eyJ1IjoidG9iaWNvcnJhZGkiLCJhIjoiY2wzZWxjNHI3MDFsNDNkbDU5ejdoMmZ4MSJ9.EmRIQFziQMHrCb4tSPbQ7Q
```
Comment (juanfopaez) [inline .env]: Add extra line here please :)

File: src/index.css [hunk 3]
```diff
@@ -43,3 +43,8 @@
 .App{
   height: 100vh;
 }
+
+.leaflet-container{
+  width: 100vw;
+  height: 100vh;
+}
```
Comment (juanfopaez) [inline src/index.css]: Add new extra line here :)

**Review by juanfopaez — CHANGES_REQUESTED** [review]

**Review by juanfopaez — APPROVED** [review]
good job 🆒

**Review by natokra — APPROVED** [review]

**Pull request merged**